src/Tribe/Cost_Utils.php

```diff
@@ -63,6 +63,10 @@ public function get_event_costs( $event ) {
 
 		$costs = tribe_get_event_meta( $event->ID, '_EventCost', false );
 
+		$costs = array_filter( (array) $costs, static function ( $cost ) {
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: e15023fee4bdc42d3a8ef6eb96e434a1</sub>